src/netconf_subscribed_notifications.h

```diff
@@ -21,13 +21,26 @@
 #include <sysrepo.h>
 
 #include "common.h"
+#include "receivers.h"
+
+/**
+ * @brief state of a configured subscribed-notifications.
+ */
+enum configured_subscription_state {
+    CONFIGURED_SUBSCRIPTION_STATE_NONE = 0,
+    CONFIGURED_SUBSCRIPTION_STATE_VALID,
+    CONFIGURED_SUBSCRIPTION_STATE_INVALID,
+    CONFIGURED_SUBSCRIPTION_STATE_CONCLUDED
+};
 
 /**
  * @brief Type of a subscribed-notifications subscription.
  */
 enum sub_ntf_type {
     SUB_TYPE_SUB_NTF,   /**< standard subscribed-notifications subscription */
-    SUB_TYPE_YANG_PUSH  /**< yang-push subscription */
+    SUB_TYPE_YANG_PUSH,  /**< yang-push subscription */
+    SUB_TYPE_CFG_SUB,  /**< configured subscriptions */
+    SUB_TYPE_CFG_YANG_PUSH,  /**< configured yang push */
```

Comment:
With the new values it would be nice to unify all the names. For example
```C
SUB_TYPE_DYN_SUB,
SUB_TYPE_DYN_YANG_PUSH,
SUB_TYPE_CFG_SUB,
SUB_TYPE_CFG_YANG_PUSH
```